### PR TITLE
Added quarter filter to UCR

### PR DESCRIPTION
**Pull request by kkrampa**
@czue 
We need this filter in a couple of enikshay reports. Probably it's better to have this filter in enikshay custom module but I didn't find way to do it. What do you think about this solution?

quarter

### CODE REVIEW

File: corehq/apps/reports_core/filters.py [hunk 1]
```diff
@@ -129,6 +129,52 @@ def filter_context(self):
         }
 
 
+class QuarterFilter(BaseFilter):
+    template = 'reports_core/filters/quarter_filter.html'
+
+    def __init__(self, name, label='Quarter', css_id=None):
```
Comment (czue) [inline corehq/apps/reports_core/filters.py]: should this be lazy translated?

**Review by czue — CHANGES_REQUESTED** [review]
This looks good to me. I think it's ok this to be here since I believe it would be generally useful for other projects/reports. Can you update the UCR docs to include this?

**Comment by kkrampa** [comment]
@czue Updated

**Review by czue — APPROVED** [review]
:+1: thanks

**Comment by kkrampa** [comment]
@czue :white_check_mark:

**Pull request merged**